src/Components/shared/ButtonLight.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { StyleSheet, Text, TouchableOpacity } from 'react-native';
+
+import theme from '../../styles/theme.style';
+
+const ButtonLight = props => {
```

Comment:
If the only difference between Button and ButtonLight is the color, we should have just one component and use a prop to pass like `light={true}` to choose when to have the light version